src/themes/plugin/index.ts

```diff
@@ -0,0 +1,98 @@
+import { handleMargin } from './margin'
+import { handlePadding } from './padding'
+
+const handle = (content: string) => {
```

Comment:
```suggestion
const transformCSSProperties = (content: string) => {
```
Probably need a better name to explain what it is doing better